include/sys/spa_impl.h

```diff
@@ -263,6 +263,7 @@ struct spa {
 	 */
 	spa_alloc_t	*spa_allocs;
 	int		spa_alloc_count;
+	uint64_t	spa_active_allocator;	/* selectable allocator */
```

Comment:
I think `uint8_t` would be better here.

Reply:
I am not sure about uint8_t, but int could fit better.  spa_set_allocator() actually uses int inside, so uint64_t here makes no sense.

Reply:
updated, thanks